inventory/components/InventoryTable.tsx

```
@@ -0,0 +1,281 @@
+                        {getValue()}
+                    </span>
+                ),
+            }),
+            columnHelper.display({
+                id: 'actions',
+                header: 'Actions',
+                cell: ({ row }) => (
+                    <button
+                        onClick={() => handleViewDetails(row.original)}
+                        className="flex items-center gap-2 px-3 py-1.5 text-sm text-blue-600 hover:text-blue-800 dark:text-blue-400 dark:hover:text-blue-300 hover:bg-blue-50 dark:hover:bg-blue-900/20 rounded-md transition-colors"
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)